lonestar/analytics/cpu/pagerank/PageRank-pull.cpp

```diff
@@ -217,32 +218,31 @@ void computePRTopological(Graph& graph) {
 
           //! New value of pagerank after computing contributions from
           //! incoming edges in the original graph.
-          float value = sum * ALPHA + (1.0 - ALPHA);
+          float value = sum * ALPHA + base_score;
           //! Find the delta in new and old pagerank values.
           float diff = std::fabs(value - sdata.value);
 
           //! Do not update pagerank before the diff is computed since
           //! there is a data dependence on the pagerank value.
           sdata.value = value;
-          max_delta.update(diff);
+          accum += diff;
         },
         galois::no_stats(), galois::steal(), galois::chunk_size<CHUNK_SIZE>(),
         galois::loopname("PageRank"));
 
-    float delta = max_delta.reduce();
-
 #if DEBUG
     std::cout << "iteration: " << iteration << " max delta: " << delta << "\n";
 #endif
 
     iteration += 1;
-    if (delta <= tolerance || iteration >= maxIterations) {
+    if (accum.reduce() <= tolerance || iteration >= maxIterations) {
       break;
     }
-    max_delta.reset();
+    accum.reset();
 
   } ///< End while(true).
 
+  std::cout << "Rounds executed: " << iteration << "\n";
```

Comment:
We print this line in stats right? If we don't, then include it in stats. Please remove this line.